CHANGES.md

```diff
@@ -40,6 +40,11 @@
 
 - Lines ending with `fmt: skip` will now be not formatted (#1800)
 
+- PR #2053: Black no longer relies on typed-ast for python versions >= 3.8
+
+- PR #2053: Python2 support is now optional, install with
+  `python3 -m pip install black[python2]`
+
```

Comment:
I like how you split up the two separate but related changes!

NIT: can we be consistent? also wording tweaks and removing unsafe advice
```suggestion
- PR #2053: Black no longer relies on typed-ast for Python 3.8 and higher

- PR #2053: Python 2 support is now optional, install with the `python2` extra to
  maintain support

```

Reply:
Is suggesting a pip command unsafe advice? the [readme](https://github.com/psf/black/#installation-and-usage) has similar commands.

Reply:
A pip command no, but `python3 -m pip install black` with `python{whatever-version}` isn't safe since it may reference a system install of Python and installing stuff into one is... a bad idea. Even `pip install black` isn't entirely safe to just copy and paste depending on how the system is set up, but it's a bit more generic to send a message that you might want to double check what Python you're about to install to. 

Also as an example, my Ubuntu 20.04 LTS installation doesn't have `pip` linked to any executable (and therefore any Python installation) so even a copy+paste would fail and not potentially break my system. I'd have to use a virtual environment to even have `pip install black` work.

Reply:
Honestly I'm not entirely sure whether this is the right stance, ex. see https://snarky.ca/why-you-should-use-python-m-pip/, just that `python3 -m pip install black` looked too copy and pastable and dangerous, it was more a knee jerk reaction than anything really. We can ignore this :)